README.md

```diff
@@ -1,72 +1,72 @@
 
-Use [Issues](https://github.com/ARM-software/CMSIS-Driver_Validation#issues-and-labels) to provide feedback and report problems for CMSIS-Driver Validation.
+## Overview
+The branch *main* of this repository contains the code of CMSIS-Driver Validation Suite/[User documentation](http://arm-software.github.io/CMSIS-Driver_Validation/main/index.html) explains the scope and the usage of the framework.
 
-**Note:** The branch *develop* of this GitHub repository reflects our current state of development and is constantly updated. It gives our users and partners contiguous access to the CMSIS development. It allows you to review the work and provide feedback or create pull requests for contributions.
+[See verified releases](https://github.com/ARM-software/CMSIS-Driver_Validation/releases) of CMSIS-Driver Validation suite in source code archives and in [CMSIS-Pack (.pack)](https://open-cmsis-pack.github.io/Open-CMSIS-Pack-Spec/main/html/index.html) format as download assets.
 
-## Directory Structure
+Feel free to raise an [issue on GitHub](https://github.com/ARM-software/CMSIS-Driver_Validation/issues) to report a misbehavior (i.e. bugs), start discussions about enhancements or create a pull request with suggested modifications.
 
-| Directory        | Content                                                   |
-| ---------------- | --------------------------------------------------------- |
-| Boards           | Driver Validation examples for a set of boards            |
-| DoxyGen          | Source of the documentation                               |
-| Include          | Header files for driver validation components             |
-| Source           | Source files for driver validation components             |
-| Tools            | SockServer and SPI_Server source and executable files     |
-| Utilities        | Utility programs                                          |
+### Supported Driver Interfaces
 
-## Generate CMSIS Pack for Release
+The CMSIS-Driver Validation provides tests for the following CMSIS-Driver interfaces:
 
-To build a complete CMSIS pack for installation the following additional tools are required:
- - **doxygen.exe**    Version: 1.9.2 (Documentation Generator)
- - **mscgen.exe**     Version: 0.20  (Message Sequence Chart Converter)
- - **7z.exe (7-Zip)** Version: 16.02 (File Archiver)
-  
-Using these tools, you can generate on a Windows PC:
- - **CMSIS Software Pack** using the batch file **gen_pack.bat** (located in ./Utilities). This batch file also generates the documentation.
- - **CMSIS Documentation** using the batch file **gen_doc.bat** (located in ./Doxygen). 
-
-## License
-
-Arm CMSIS-Driver Validation is licensed under Apache-2.0.
-
-## Contributions and Pull Requests
-Contributions are accepted under Apache-2.0. Only submit contributions where you have authored all of the code.
+- **CAN** - Controller Area Network interface driver.
```

Comment:
I would add additional information like following:
Extensive tests with available servers:
- **SPI** - Serial Peripheral Interface driver.
- **USART** - Universal Synchronous and Asynchronous Receiver/Transmitter interface driver.
- **WiFi** - Wireless Fidelity Interface module/shield driver.
Basic tests:
- **Ethernet** - Ethernet MAC and PHY peripheral interface driver.
- **I2C** - Inter-Integrated Circuit multi-master serial single-ended bus interface - driver.
- **MCI** - Memory Card Interface driver for SD/MMC memory.
- **USB Device** - Universal Serial Bus Device interface driver.
- **USB Host** - Universal Serial Bus Host interface driver.